tests/e2e/e2e_test.go

```diff
@@ -29,19 +28,155 @@ import (
 	"github.com/osmosis-labs/osmosis/osmoutils/osmoassert"
 	appparams "github.com/osmosis-labs/osmosis/v16/app/params"
 	v16 "github.com/osmosis-labs/osmosis/v16/app/upgrades/v16"
+	"github.com/osmosis-labs/osmosis/v16/tests/e2e/configurer/chain"
 	"github.com/osmosis-labs/osmosis/v16/tests/e2e/configurer/config"
 	"github.com/osmosis-labs/osmosis/v16/tests/e2e/initialization"
 	clmath "github.com/osmosis-labs/osmosis/v16/x/concentrated-liquidity/math"
 	cltypes "github.com/osmosis-labs/osmosis/v16/x/concentrated-liquidity/types"
 )
 
-// Reusable Checks
+// TODO: Find more scalable way to do this
+func (s *IntegrationTestSuite) TestAllE2E() {
+	// Zero Dependent Tests
+	s.T().Run("CreateConcentratedLiquidityPoolVoting_And_TWAP", func(t *testing.T) {
```

Comment:
nit: I feel like these would benefit from writing them in a struct similar to how we do with table tests. So it becomes easier to skim